### ISSUE TITLE
Check whether GCM respect the ignore annotation

### ISSUE THREAD

**Issue by vpnachev** [gardener/dashboard#386]
The gardener controller manager could be configured[1] to not respect the annotation `shoot.garden.sapcloud.io/ignore`, which means that it will reconcile the shoot cluster even if the annotation is set properly. The dashboard itself is just checking the annotation to determine whether the reconciliation is enabled or not. 

[1] https://github.com/gardener/gardener/blob/2f79201d69dc27ea738d70785a0aa5223bf3481a/pkg/controllermanager/apis/config/v1alpha1/types.go#L175-L178

**Comment by grolu** [comment]
@petersutter we already discussed this. If the dashboard has the information if reconciliation is enabled in the gardener controller manager, we could also expose the feature in the ui (for operators only)

**Comment by vpnachev** [comment]
I was thinking about a possible implementation.
To the best of my knowledge the gardener does not expose it's configuration, so the dashboard cannot read that config and act upon it. Of course, the configuration could be a config map, but it is not mandatory.

So what could be done is to implement a `flag`/`env var`/`field in the dashboard config file`  that could be configured just like the gardener is configured. Then whoever deploys the gardener and the dashboard will be responsible to keep both configurations in sync.

**Comment by grolu** [comment]
@vpnachev yes we had to do this in other cases, too. Latest example is determining gardener admission controller `candidateDeterminationStrategy` configuration (https://github.com/gardener/dashboard/pull/370).
It is not ideal but the only way to do it I guess.

**Comment by vpnachev** [comment]
Yes, it looks like the right solution is to do something similar as for the `seedCandidateDeterminationStrategy`.

**Comment by petersutter** [comment]
https://github.com/gardener/dashboard/pull/628#issuecomment-600590445 is prerequisite
